setup.py

```diff
@@ -37,8 +37,11 @@
         exclude=["*.tests", "*.tests.*", "tests.*", "tests"]),
     package_data={'': (['*.cpp'] + runtime_files)},
     install_requires=[
-        'dace@git+https://github.com/spcl/dace.git@40d735', 'onnx == 1.8.0',
-        'torch', 'protobuf == 3.19', 'dataclasses; python_version < "3.7"',
+        'dace == 0.13.3',
+        'onnx == 1.7.0',  # we support opset v12
```

Comment:
did we downgrade ONNX here?

Reply:
ONNX 1.8's built in tests are the tests for opset 13, but we implement opset 12